examples/sampling/graphbolt/node_classification.py

```diff
@@ -312,6 +312,7 @@ def train(args, graph, features, train_set, valid_set, num_classes, model):
             y = data.labels
 
             y_hat = model(data.blocks, x)
+            y_hat = y_hat[data.compacted_seeds]
```

Comment:
Why do we need to add this?

Reply:
When `seeds` go through `unique_and_compact` on cuda mode, it will be sorted in ascending order which may be different from original order and may mismatch with `labels`. So, we need to use `compacted_seeds` to restore it to its original order.

Reply:
Does this caused by your refactor? Or it was an existing bug?